administration/src/graphql/regions/getRegions.graphql

```diff
@@ -1,5 +1,5 @@
-query getRegions {
-    regions {
+query getRegions($project: String!) {
```

Comment:
Can we already remove the getRegions endpoint from the backend? We can break the API as we want on the whitelabel branch.

Reply:
Yep, we can remove all deprecated endpoints after this.